src/module/apps/compendium-browser/index.ts

```diff
@@ -81,6 +91,131 @@ class PackLoader {
             record.img = actorArt ?? record.img;
         }
     }
+
+    #getAllowedSources() {
```

Comment:
Add return types to all these signatures

Reply:
I did that, rebased and also fixed a smaller issue I found while testing.